data/users/test/test_exceptions.py

```
@@ -0,0 +1,273 @@
+
+from data.users.externalldap import (
+    LDAP_CONNECTION_ERRORS,
+    LDAPUsers,
+    _log_ldap_error,
+    logger,
+)
+
+
+class ExceptionLogHandler(logging.StreamHandler):
+    on_same_line = False
+    content = []  # type: List[str]
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)